app/views/articles/_index_default.html.erb

```diff
@@ -31,17 +31,24 @@
   </dl>
 <% end %>
 
-<% if document.access_panels.online? && !document.eds_restricted? %>
+<% if !document.eds_restricted? %>
   <ul class='document-metadata dl-horizontal dl-invert'>
-    <% document.access_panels.online.links.each do |link| %>
-      <li>
-        <% if link.ill? -%>
-          <%= link_to(link.text, link.href, class: 'sfx') %>
-        <% else %>
+    <% if document.access_panels.online? %>
+        <% document.access_panels.online.links.each do |link| %>
+          <li>
+            <% if link.ill? -%>
+              <%= link_to(link.text, link.href, class: 'sfx') %>
+            <% else %>
+              <span class="online-label">Full text</span>
+              <%= link_to(link.text, link.href) %>
+            <% end %>
+          </li>
+          <% end %>
+        <% elsif document['eds_html_fulltext_available'].present? %>
+        <li>
           <span class="online-label">Full text</span>
```

Comment:
The indentation here is making this a bit difficult to follow, would you mind fixing that up in this commit?